mmcv/ops/nms.py

```diff
@@ -325,11 +325,16 @@ def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
         dets, keep = nms_op(boxes_for_nms, scores, **nms_cfg_)
         boxes = boxes[keep]
         # -1 indexing works abnormal in TensorRT
-        # This assumes `dets` has 5 dimensions where
+
+        # This assumes `dets` has arbitrary dimensions where
         # the last dimension is score.
+        # Currently it supports bounding boxes [x1, y1, x2, y2, score] and
+        # rotated boxes [cx, cy, w, h, angle_radian, score].
         # TODO: more elegant way to handle the dimension issue.
         # Some type of nms would reweight the score, such as SoftNMS
-        scores = dets[:, 4]
+
+        score_dim = dets.size(1) - 1
+        scores = dets[:, score_dim]
```

Comment:
`dets[:, -1]`

Reply:
```
# -1 indexing works abnormal in TensorRT
```

But the comment says it can cause abnormal behavior. Is it okay to use -1?
I didn't test it yet.

Reply:
I'll check it.

Reply:
> Check what `# -1 indexing works abnormal in TensorRT` means. @RunningLeon Then decide that can we change to `dets[:, -1]`

-1 indexing is OK, because mmcv.ops.nms is replace by the `dummy_nms_for_onnx` while in exporting to ONNX for supported models.
https://github.com/open-mmlab/mmdetection/blob/5e246d5e3bc3310b5c625fb57bc03d2338ca39bc/mmdet/models/dense_heads/base_dense_head.py#L521

Reply:
I am trying to edit it as  -1 indexing. But, can anyone suggest a comment about it?
Is it okay to delete the mention about -1 indexing TensorRT?
Or is it better to add a mention the `dummy_nms_for_onnx`?